runtime.go

```diff
@@ -278,13 +361,45 @@ func (self *_runtime) toValue(value interface{}) Value {
 			case reflect.Func:
 				// TODO Maybe cache this?
 				return toValue_object(self.newNativeFunction("", func(call FunctionCall) Value {
-					in := make([]reflect.Value, len(call.ArgumentList))
+					argsCount := len(call.ArgumentList)
+					in := make([]reflect.Value, argsCount)
 					t := value.Type()
+					callSlice := false
+					paramsCount := t.NumIn()
+					lastParam := paramsCount - 1
+					lastArg := argsCount - 1
+					isVariadic := t.IsVariadic()
 					for i, value := range call.ArgumentList {
-						in[i] = safeNumericConvert(t, i, value.export())
+						if isVariadic && i == lastArg && argsCount == paramsCount {
+							// Variadic functions last parameter and parameter numbers match incoming args
+							paramType := t.In(lastArg)
+							val := reflect.ValueOf(value.export())
+							callSlice = callSliceRequired(paramType, val)
+							if callSlice {
+								in[i] = callParamConvert(reflect.ValueOf(value.export()), paramType)
+								continue
+							}
+						}
+						var paramType reflect.Type
```

Comment:
Could do with a new line here to separate out the blocks of logic. Also, in the case of callSlice not being true, you are redeclaring this variable (while it's declared previous in a different scope, it's possibly a source of confusion - could you just declare it once at the start of the loop or do you need it to be the zero-value for reflect.Type for some reason?